### ISSUE TITLE
Information request: pkg commands

### ISSUE THREAD

**Issue by Wolfram-Wolf** [termux/termux-app#522]
Why are the _pkg_ commands provided when one has the _apt_ commands?

**Comment by Grimler91** [comment]
`more ``which pkg``` gives:

```
...
CMD="$1"
shift 1

case "$CMD" in
        f*) dpkg -L $@;;
        h*) show_help;;
        add|i*) apt update; apt install $@;;
        list-a*) apt list $@;;
        list-i*) apt list --installed $@;;
        re*) apt install --reinstall $@;;
        se*) apt update; apt search $@;;
        sh*) apt show $@;;
        un*|rem*|rm|del*) apt remove $@;;
        up*) apt update; apt full-upgrade;;
        *) echo "Unknown command: '$CMD' (run 'pkg help' for usage informatio
n)";;
esac
```

Basically it is a wrapper around apt which runs `apt update` before trying to install something.
